js/src/launchAsync.ts

```diff
@@ -13,18 +14,30 @@ type Message = {
     launchToPostMessageSentDurationInMs: number;
 };
 
+type LaunchResponseMessage = {
```

Comment:
do you need to define this type LaunchResponseMessage ?

I think that your code would work without it